src/ui/class-post-submitbox.php

```diff
@@ -123,4 +153,109 @@ public function add_rewrite_and_republish_post_button( $post = null ) {
 			}
 		}
 	}
+
+	/**
+	 * Changes the 'Publish' copies in the submitbox to 'Republish' if a post is intended for republishing.
+	 *
+	 * @param string $translation The translated text.
+	 * @param string $text        The text to translate.
+	 *
+	 * @return string The to-be-used copy of the text.
+	 */
+	public function change_republish_strings_classic_editor( $translation, $text ) {
+		if ( ! $this->should_change_rewrite_republish_copy( \get_post() ) ) {
+			return $text;
+		}
+
+		if ( $text === 'Publish' ) {
+			return \__( 'Republish', 'duplicate-post' );
+		}
+
+		return $translation;
+	}
+
+	/**
+	 * Changes the 'Schedule' copy in the submitbox to 'Schedule republish' if a post is intended for republishing.
+	 *
+	 * @param string $translation The translated text.
+	 * @param string $text        The text to translate.
+	 * @param string $context     Context information for the translators.
+	 *
+	 * @return string The to-be-used copy of the text.
+	 */
+	public function change_schedule_strings_classic_editor( $translation, $text, $context ) {
+		if ( ! $this->should_change_rewrite_republish_copy( \get_post() ) ) {
+			return $text;
+		}
+
+		if ( $text === 'Schedule' ) {
+			return \__( 'Schedule republish', 'duplicate-post' );
```

Comment:
The other two occurrences of `Schedule Republish` have an uppercase `R`. For consistency, they should all use the same casing. Note: While WordPress traditionally uses title case, at Yoast we generally prefer sentence case. Personally, I have no strong preferences in this case but it would be good to make them consistent. See also the related test.